cpp/src/phonenumbers/base/basictypes.h

```diff
@@ -5,13 +5,13 @@
 #ifndef I18N_PHONENUMBERS_BASE_BASICTYPES_H_
 #define I18N_PHONENUMBERS_BASE_BASICTYPES_H_
 
-#include <limits.h>         // So we can set the bounds of our types
-#include <stddef.h>         // For size_t
-#include <string.h>         // for memcpy
+#include <climits>  // So we can set the bounds of our types
```

Comment:
For such files as this one, that are copied verbatim from other open source projects, I think it would be better to not touch them at all, for it makes maintenance easier if they're unchanged so that newer versions can be easily copied from their projects of origin.

Reply:
Are this the only changes I should leave? I'll like to know the files this applies to. Thanks.

Reply:
> Are this the only changes I should leave?

I don't know. Sorry.

Reply:
I'll go through the files and keep legacy changes. Thanks.